tool/jt.rb

```diff
@@ -768,6 +768,7 @@ def help
                                               it is also linked in your ruby manager (if found) under the same name,
                                               by default it is the name of the mx env file,
                                               the named build stays until it is rebuilt or deleted manually
+              --new-hash                      update the Git hash in RUBY_DESCRIPTION
```

Comment:
`update the git commit hash in RUBY_DESCRIPTION`

Reply:
What other kinds of hash does Git have?

Reply:
A file/tree hash for instance. sha1 hashes are used for many different things in git.
This information is "which commit was used" not "some hash".

Reply:
Are we still blocked on this?

Reply:
I'll change it ...